### PR TITLE
Several comments and minor changes.

### PR DESCRIPTION
**Pull request by bbb999**
Several comments and minor changes.

### CODE REVIEW

File: PProPanel/jsx/Premiere.jsx [hunk 1]
```diff
@@ -248,19 +236,18 @@ $._ext_PPRO={
         
         if (app.project != null) {
 
-        // Find or create a target bin.
-
+	        // If panel can't find the target bin, it creates it.
         var nameToFind = 'Targeted by PProPanel import';
```
Comment (lesavage) [inline PProPanel/jsx/Premiere.jsx]: are you wanting to create a new variable or overwrite the supplied parameter?

Comment (bbb999) [inline PProPanel/jsx/Premiere.jsx]: Yes, if there is not yet a bin with 'nameToFind', I want to create it.

Comment (lesavage) [inline PProPanel/jsx/Premiere.jsx]: But you don't check if nameToFind exists before updating it...?

Comment (lesavage) [inline PProPanel/jsx/Premiere.jsx]: Argh ignore me. My fault! I thought the code was part of function searchForBinName and I could not understand why you were not reassigning nameToFind. I did not see the importFiles function definition.

**Pull request merged**